metricbeat/module/jolokia/jmx/_meta/docs.asciidoc

```diff
@@ -1,58 +1,67 @@
-This is the `jmx` metricset of the Jolokia module.
+The `jmx` metricset collects metrics from
+https://jolokia.org/reference/html/agents.html[Jolokia agents].
 
 [float]
 === Features and configuration
 Tested with Jolokia 1.3.4.
 
-Metrics to be collected from each Jolokia instance are defined in the mapping section with an MBean ObjectName and
-an array of Attributes to be requested with Elastic field names under which the return values should be saved.
+To collect metrics from a Jolokia instance, define a mapping section that
+specifies an MBean `ObjectName` followed by an array of attributes to fetch. For
+each attribute in the array, specify the Elastic field name where the
+returned value will be saved. 
 
-For example: to get the "Uptime" attribute from the "java.lang:type=Runtime" MBean and map it to something like
-"uptime" (actually "jolokia.jmx.uptime", the prexif is added by beats framework) you have to configure following
-mapping:
+For example, to get the `Uptime` attribute from the `java.lang:type=Runtime`
+MBean and map it to an event field called `jolokia.jmx.uptime`, configure the
+following mapping:
 
 [source,yaml]
----
+----
 - module: jolokia
   metricsets: ["jmx"]
   hosts: ["localhost:8778"]
-  namespace: "metrics"
+  namespace: "metrics" <1>
   jmx.mappings:
     - mbean: 'java.lang:type=Runtime'
       attributes:
         - attr: Uptime
-          field: uptime
-          event: uptime
+          field: uptime <2>
+          event: uptime <3>
       target:
           url: "service:jmx:rmi:///jndi/rmi://targethost:9999/jmxrmi"
           user: "jolokia"
           password: "s!cr!t"
----
+----
+<1> The `namespace` setting is required.
+<2> This field will be called `jolokia.jmx.uptime` in the output event because
```

Comment:
Actually, after configuring the namespace as `metrics`, the field will be called `jolokia.metrics.uptime`. Probably the namespace was added to have events with the monitored service in the name (e.g. if monitoring tomcat with jolokia having the metrics under `jolokia.tomcat`). Maybe it worths to add a line about the use of namespace.